jaeger-core/src/main/java/io/jaegertracing/Configuration.java

```diff
@@ -313,8 +314,7 @@ public static SamplerConfiguration fromEnv() {
           .withManagerHostPort(getProperty(JAEGER_SAMPLER_MANAGER_HOST_PORT));
     }
 
-    // for tests
-    Sampler createSampler(String serviceName, Metrics metrics) {
+    public Sampler createSampler(String serviceName, Metrics metrics) {
```

Comment:
Again, `Metrics` is internal so if this method is still required then I guess it should be `MetricsFactory`?

Reply:
The fact that it previously had a comment `for tests`, wondering whether should be public?

Reply:
Tests are the first consumers of an API: if tests need to create a sampler, chances are high that a consumer will need as well.